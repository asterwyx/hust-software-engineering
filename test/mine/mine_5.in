preset user AQ

preset userloc A 62 0
preset userloc Q 62 0

step 6
step 7

dump