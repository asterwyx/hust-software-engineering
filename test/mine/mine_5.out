user AQ

fund A 10000
fund Q 10000

credit A 80
credit Q 20

userloc A 68 0
userloc Q 69 0

nextuser A